### DISCUSSION TITLE
Error: A UPnP device has been found,  but it wasn't recognized as an IGD

### DISCUSSION THREAD

**Discussion by nikolanii** [LizardByte/community#91] [Sunshine] [answered]
This error pops up when I start Sunshine, can't fix it. UPNP is enabled in my router settings, Sunshine was added to the firewall.

How can I solve this?
Thanks in advance!

**Comment by ImJoosan** [comment, marked as answer]
This error shouldn't impact anything. I get the same error whenever I use sunshine as well.
